portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,39 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    //Create an ArrayList of strings and add three messages
+    ArrayList<String> messages = new ArrayList<String>();
+    messages.add("Hello Friends");
+    messages.add("Howdy Partner");
+    messages.add("Heyo Bucko");
+
+    // Convert messages to JSON
+    String json = convertToJson(messages);
+
+    // Send JSON to /data
     response.setContentType("text/html;");
-    response.getWriter().println("Hello CJ!");
+    response.getWriter().println(json);
+  }
+
+  private String convertToJson(ArrayList<String> array) {
+    String json = "{";
```

Comment:
What happens when the array has less than 3 items?

It would be safer/length-agnostic if this iterated through the array instead.

Reply:
So with the length, it was told in the tutorial we would be doing hard-coded for this tutorial and in an upcoming one use user input. So for the user input, I will definitely go for iterating over the array.

Reply:
Okay, sounds good then.